### PR TITLE
AbsorbBar PreView

### PR DESCRIPTION
**Pull request by ChargeIn**
The first attempt for fixing the AbsorbBar

### CODE REVIEW

File: UnitFrames/CandyUI_UnitFrames.lua [hunk 1]
```diff
@@ -1582,4 +1585,4 @@ end
 -- CandyUI_UnitFrames Instance
 -----------------------------------------------------------------------------------------------
 local CandyUI_UnitFramesInst = CandyUI_UnitFrames:new()
-CandyUI_UnitFramesInst:Init()
```
Comment (ChargeIn) [inline UnitFrames/CandyUI_UnitFrames.lua:1585]: No clue what happend there

**Comment by ChargeIn** [comment]
Since the diff is still broken :/ here the updates on the dashbar:
i only added 1 new Event:
Apollo.RegisterEventHandler("SprintEnergyUpdated", "OnSprintEnergyUpdated", self)

calling the function to updated the sprint bar:
function CandyUI_Dash:OnSprintEnergyUpdated(nRunCurr, nRunMax)
	local bar = self.wndMain:FindChild("SprintBar")
	
	bar:SetMax(nRunMax)
	bar:SetProgress(nRunCurr)
	SendVarToRover("testing var: tItem", nRunMax)
	if (nRunCurr < 170) then 
		bar:SetBGColor("red")
		return
	end
	if (nRunCurr < 340) then 
		bar:SetBGColor("xkcdAmber")
		return
	end
	bar:SetBGColor("ChatGuild")
	
end

**Comment by coding-red-panda** [comment]
Is this ready to be merged?

**Comment by ChargeIn** [comment]
yes , the sprintmeter works fine no errors etc, the absorb bar looks better than i thought

**Pull request merged**